<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecoverPassword</class>
 <widget class="QDialog" name="RecoverPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nome do usu√°rio:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_nome"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bt_buscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pergunta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_pergunta">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Resposta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_resposta"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bt_confirmar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Confirmar</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
